apps/cookbook/src/app/examples/card-example/examples/card-example.shared.scss

```
@@ -79,3 +84,27 @@ button {
     transform: scale(0.95);
   }
 }
+
+.variant-card-container {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 16px;
```

Replace magic string with defined size:
```suggestion
  gap: utils.size('s');
```